trace86/jhu-aai
# Add ai vs human play

I really need someone to review this PR and play games because I may have introduced a lack of randomness since the game always starts in 3x3 in the exact same spot, but I am not sure how this is because I am sending a random value initially. Is this what was happening before or is this something I introduced?

To test:
* Checkout this branch `add_ai_vs_human_play`
* Copy `_TEMPenv` into your `.env`
* Play as human and as attacker or defender by updated in the `.env` file
 ```AI_VS_HUMAN=0 # python-dotenv does not parse boolean, so 1 == True and 0 == False
HUMAN_PLAYS=2 # 1 == ATTACK(X), 2 == DEFENSE(O)
```

* Test that AI vs AI continues to work
* Test that randomness of game is as expected
